src/ru/skypro/Main.java

```diff
@@ -3,6 +3,108 @@
+
+        float[] myVariable = {0.25f, 0.75f, 1.1f};
+
+        // TASK 2
+        for (int i = 0; i < numbers.length; i++) {
+            if (i != numbers.length - 1) {
+                System.out.print(numbers[i] + ", ");
+            } else
+                System.out.print(numbers[i]);
+
+        }
+        System.out.println();
+
+        for (int i = 0; i < fractions.length; i++) {
+            if (i != fractions.length - 1) {
+                System.out.print(fractions[i] + ", ");
+            } else
+                System.out.print(fractions[i]);
+        }
+        System.out.println();
+
+        for (int i = 0; i < myVariable.length; i++) {
+            if (i != myVariable.length - 1) {
+                System.out.print(myVariable[i] + ", ");
+            } else
+                System.out.print(myVariable[i]);
+        }
+
+        System.out.println();
+
+        // TASK 3
+
+        for (int i = numbers.length - 1; i >= 0; i--) {
+            if (i != 0) {
+                System.out.print(numbers[i] + ", ");
+            } else
+                System.out.print(numbers[i]);
+        }
+        System.out.println();
+
+        for (int i = fractions.length - 1; i >= 0; i--) {
+            if (i != 0) {
+                System.out.print(fractions[i] + ", ");
+            } else
+                System.out.print(fractions[i]);
+        }
+        System.out.println();
+
+        for (int i = myVariable.length - 1; i >= 0; i--) {
+            if (i != 0) {
+                System.out.print(myVariable[i] + ", ");
+            } else
+                System.out.print(myVariable[i]);
+        }
+
+        System.out.println();
+        //TASK 4
+
+        for (int i = 0; i < numbers.length; i++) {
+            if (i != numbers.length - 1 && numbers[i] % 2 == 0) {
+                System.out.print(numbers[i] + ", ");
+            } else if (i != numbers.length - 1 && numbers[i] % 2 != 0) {
+                numbers[i] += 1;
+                System.out.print(numbers[i] + ", ");
+            } else if (i == numbers.length - 1 && numbers[i] % 2 != 0) {
+                numbers[i] += 1;
+                System.out.print(numbers[i]);
+            } else
+                System.out.print(numbers[i]);
+        }
+        System.out.println();
+        //ИЛИ
+        for (int i = 0; i < numbers.length; i++) {
+            if (numbers[i] % 2 != 0) {
+                numbers[i] += 1;
+            } else if (i != numbers.length - 1) {
+                System.out.print(numbers[i] + ", ");
+            } else {
+                System.out.print(numbers[i]);
+            }
+        }
+        System.out.println();
+        //ИЛИ
+        for (int i = 0; i < numbers.length; i++) {
+            if (numbers[i] % 2 != 0) {
+                numbers[i] += 1;
+            } if (i != numbers.length - 1) {
+                System.out.print(numbers[i] + ", ");
+            } else {
+                System.out.print(numbers[i]);
+            }
```

Comment:
визуально кажется что это все одно большое условие, поэтому стоит их более явно разделить
```suggestion
            if (numbers[i] % 2 != 0) {
                numbers[i] += 1;
            } 
            if (i != numbers.length - 1) {
                System.out.print(numbers[i] + ", ");
            } else {
                System.out.print(numbers[i]);
            }
```